user-guide/content-curation/external-collection-indexing/arcgis-server-integration.qmd

```diff
@@ -0,0 +1,242 @@
+---
+title: ArcGIS Server Integration
+subtitle: Indexing ArcGIS Server services using pyarc2stac for STAC collection creation
+---
+
+The [pyarc2stac](https://github.com/NASA-IMPACT/pyarc2stac) library provides seamless integration between ArcGIS Server services and STAC collections, enabling VEDA to index and access geospatial data from ArcGIS Server endpoints without requiring data migration.
+
+## Overview
+
+ArcGIS Server hosts various types of geospatial services that can be converted to STAC collections for integration with VEDA. The pyarc2stac library automatically extracts metadata from ArcGIS services and generates STAC-compliant collection documents.
+
+## Supported ArcGIS Services
+
+| Service Type | Use Cases |
+|--------------|-----------|
+| **ImageServer** | Satellite imagery, raster analysis, multidimensional data |
+| **MapServer** | Visualized map services, layer-based data |
+| **FeatureServer** | Vector data, administrative boundaries, point datasets |
+
+## Key Features
+
+- **Automatic Metadata Extraction**: Pulls spatial extent, temporal coverage, and descriptive metadata
+- **WMS Integration**: Generates WMS links for visualization in VEDA Dashboard
+- **Datacube Support**: Handles multidimensional ImageServer data with proper datacube extensions
+- **STAC Compliance**: Generates valid STAC collections following specification standards
+- **Render Configuration**: Automatically configures visualization parameters
+
+## Architecture
+
+```{mermaid}
+graph LR
+    A[ArcGIS Server] --> B[pyarc2stac Library]
+    B --> C[STAC Collection]
+    C --> D[VEDA Catalog]
+    
+    subgraph "ArcGIS Services"
```

Comment:
For some reason the subgraphs in the architecture diagrams for both integration diagrams are hard to read--might be my browser settings?

<img width="1219" height="526" alt="Image" src="https://github.com/user-attachments/assets/35b89d83-2aa9-41f6-bba1-fbd378b314e2" />